examples/plot_newfitting_comparison_oldnew.py

```diff
@@ -0,0 +1,221 @@
+    s = LTEModel(factory, fit_parameters, fixed_parameters)
+    # we could also have added a fittable parameter, such as an offset,
+    # or made the slit width a fittable parameter.
+    # ... any paramter in model_input will be fitted.
+    # s.offset(model_input["offset"], 'nm')
+    s.apply_slit(1.4, "nm")
+    return s.take("radiance").normalize()
+
+
+sf = SpectrumFactory(
+    wlmin * u.nm,
+    wlmax * u.nm,
+    wstep=0.001,  # cm-1
+    pressure=1 * 1e-3,  # bar
+    cutoff=1e-25,
+    isotope="1,2",
+    path_length=10,  # cm-1
+    mole_fraction=1,
+    truncation=1,  # cm-1
+)
+sf.warnings["MissingSelfBroadeningWarning"] = "ignore"
+sf.warnings["HighTemperatureWarning"] = "ignore"
+sf.load_databank("HITRAN-CO2-TEST")
+
+begin_time_mark = time.time()
+
+s_best, best = sf.fit_spectrum(
+    s_exp.take("radiance"),
+    model=LTEModel_withslitnorm,
+    fit_parameters={
+        "Tgas": 300,
+        # "offset": 0
+    },
+    bounds={
+        "Tgas": [300, 2000],
+        # "offset": [-1, 1],
+    },
+    plot=True,
+    solver_options={
+        "maxiter": 15,  # 👈 increase to let the fit converge
+        "ftol": 1e-15,
+    },
+    verbose=2,
+)
+
+end_time_mark = time.time()
+
+plot_diff(s_exp, s_best)
+
+# Information to report later in comparison result
+oldfitting_residual = best.fun
+oldfitting_loops = best.nfev
+oldfitting_time = end_time_mark - begin_time_mark
+
+# ------------------------------------------------------------------------------------------------------------ #
+
+
+# -------------------------------------------- NEW FITTING MODULE -------------------------------------------- #
+
+
+# ------------------------------------ Step 1. Load experimental spectrum ------------------------------------ #
+
+
+# Load an experimental spectrum. You can prepare yours, or fetch one of them in the radis/test/files directory.
+my_spec = getTestFile("CO2_measured_spectrum_4-5um.spec")
+s_experimental = load_spec(my_spec).offset(-0.2, "nm")
+
+
+# ------------------------------------ Step 2. Fill ground-truths and data ------------------------------------ #
+
+
+# Experimental conditions which will be used for spectrum modeling. Basically, these are known ground-truths.
+experimental_conditions = {
+    "molecule": "CO2",  # Molecule ID
+    "isotope": "1,2",  # Isotope ID, can have multiple at once
+    "wmin": 4167,  # Starting wavelength/wavenumber to be cropped out from the original experimental spectrum.
+    "wmax": 4180,  # Ending wavelength/wavenumber for the cropping range.
+    "wunit": "nm",  # Accompanying unit of those 2 wavelengths/wavenumbers above.
+    "mole_fraction": 1,  # Species mole fraction, from 0 to 1.
+    "pressure": 1 * 1e-3,  # Partial pressure of gas, in "bar" unit.
+    "path_length": 10,  # Experimental path length, in "cm" unit.
+    "slit": "1.4 nm",  # Experimental slit, must be a blank space separating slit amount and unit.
+    "wstep": 0.001,  # Resolution of wavenumber grid, in cm-1.
```

Comment:
using "auto" should be best, no ? What's the default ?

Reply:
(btw if you want to compare Performances with the other method; it should be the same inputs)

Reply:
Because originally in the [1-temperature fitting example](https://radis.readthedocs.io/en/latest/auto_examples/plot_1T_fit.html#sphx-glr-auto-examples-plot-1t-fit-py) they use `wstep=0.001`. 

I am comparing the performance between 2 fitting machines, so I must set completely the same experimental inputs.

Reply:
Don't forget to add the performance comparison in the PR test

Reply:
And (see my other comment) --> delete wstep ; keep the default; and make sure to use wstep=0.001 but only for the performance comparison. Let's clean the example --> minimal is best

Reply:
resolve comment if done